### ISSUE TITLE
[Write] Docs for Utilities - Property Utilities 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2204]
Would be nice to have in SB. can we make this possible? 

**User Story:**
As a Spark we want to write Property Utilities Docs in Sharepoint Word so that we can get reviews completed and then once reviews have been completed, we can create PRs for Gatsby and Storybook and users will see helpful documentation on the New Doc Site.

**Notes:**
- this is dependent on #2349 and the result of #2349 (or we write it in word)
- work with Dave on this, Dave will review content
- Property Utilities will display both Single and Multi Revised Content
https://sparkdesignsystem.com/patterns/utilities/multi-property
https://sparkdesignsystem.com/patterns/utilities/single-property
- Content will live in Gatsby??

**AC:**
- Review/Edit/Revise Current docs content for Single and Multi
- Write detailed Documentation in Sharepoint Word
